apps/src/templates/SpeedSlider.jsx

```diff
@@ -0,0 +1,233 @@
+var React = require('react');
+var color = require('../color');
+var Radium = require('radium');
+var dom = require('../dom');
```

Comment:
I'm not getting errors when I `import dom from 'dom.js'` so I'll go ahead and move all of them over.